quickstart.sh

```diff
@@ -27,6 +27,8 @@ cassandraFolder=_cassandra
 envFile=.env
 configFile=_config/config.php
 httpPort=8000
+cassandraDeprecationVersion="9.0.0"
```

Comment:
@shubham-kickdrumtech similarly to what I commented on the other line, let's add a comment to make it clear that this value will need to be modified later.

CC @gneb

Reply:
@damianoferrari I have added the comment.